internal/vercheck/notification.tmpl

```diff
@@ -0,0 +1,19 @@
+{{- with .Advisory -}}
+SECURITY ADVISORY
+{{ .Text }}
+{{- end }}
+{{- with .Latest -}}
+A new version of Atlas Provider is available ({{ .Version }}){{ with .Link }}: {{ . }}{{ end }}
+{{- with .Summary }}
+{{ . }}
+{{- end }}
+
+terraform {
```

Comment:
Great idea !

Reply:
Because I saw the link was not helpful for the upgrade, it's just sharing the release note.